src/json-validation.ts

```diff
@@ -0,0 +1,18 @@
+import * as fs from "fs";
+
+import Ajv from "ajv";
+
+type Schema = "RepoArray" | "Instructions";
+
+export function validateObject<T>(obj: unknown, schema: Schema): T {
```

Comment:
Thanks. That's a great suggestion for making the types stronger, and reading the schemas by imports rather than reading the file at runtime also looks much better.

I've gone slightly further too and made it only compile each schema once and only when needed. The recommendation from Ajv seems to be to either compile everything once at the start, or compile when needed but memoize the result.